recipes/conversation_view/feed_item_row/feed_item_row.vue

```diff
@@ -157,11 +157,21 @@ export default {
       default: false,
     },
 
-    // TODO: implement as part of DT-1157
-    // state: {
-    //   type: String,
-    //   default: 'normal',
-    // },
+    /**
+     * state for the feed item row. Can be default, searched & error
+     */
+    state: {
+      type: String,
+      default: 'NORMAL',
```

Comment:
Need to add a constant for the allowed state values, and constrain this prop to only accept the allowed values.